### PR TITLE
Upgrade to Rails 6.1

### PR DESCRIPTION
**Pull request by raccube**

### CODE REVIEW

File: config/initializers/assets.rb [hunk 1]
```diff
@@ -1,22 +1,12 @@
...
 Rails.application.config.assets.version = '1.0'
 
-# Add additional assets to the asset load path
+# Add additional assets to the asset load path.
 # Rails.application.config.assets.paths << Emoji.images_path
 
 # Precompile additional assets.
-# application.js, application.css, and all non-JS/CSS in app/assets folder are already added.
-Rails.application.config.assets.precompile += %w( moderation.js )
-Rails.application.config.assets.precompile += %w( i18n.js )
```
Comment (nilsding) [inline config/initializers/assets.rb:22]: this is not required anymore because of webpacker, right?

File: db/schema.rb [hunk 2]
```diff
@@ -180,7 +180,7 @@
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
     t.index ["name", "resource_type", "resource_id"], name: "index_roles_on_name_and_resource_type_and_resource_id"
-    t.index ["resource_type", "resource_id"], name: "index_roles_on_resource_type_and_resource_id"
+    t.index ["resource_type", "resource_id"], name: "index_roles_on_resource"
```
Comment (nilsding) [inline db/schema.rb:183]: why did this index name change?

File: spec/helpers/bootstrap_helper_spec.rb [hunk 3]
```diff
@@ -88,9 +88,9 @@
 
   describe "#time_tooltip" do
     it 'should return a tooltip with proper time values' do
-      Timecop.freeze(Time.local(1984)) do
+      Timecop.freeze(Time.utc(1984)) do
```
Comment (nilsding) [inline spec/helpers/bootstrap_helper_spec.rb:91]: literally 1984

File: config/initializers/assets.rb (hunk 1 above)
Comment (raccube) [reply config/initializers/assets.rb:22]: If I'm not mistaken we still use the asset pipeline for i18n. I'm also fairly certain rails admin still uses it

File: spec/helpers/bootstrap_helper_spec.rb (hunk 3 above)
Comment (raccube) [reply spec/helpers/bootstrap_helper_spec.rb:91]: wortwörtlich 1984

File: config/initializers/assets.rb (hunk 1 above)
Comment (pixeldesu) [reply config/initializers/assets.rb:22]: Yes:
* `moderation.js` as an entrypoint doesn't exist anymore
* `i18n.js` is not used as an exported asset anymore either

File: db/schema.rb (hunk 2 above)
Comment (raccube) [reply db/schema.rb:183]: Just cleared out my db and rerun migrations; this is how it generated.

Comment (nilsding) [reply db/schema.rb:183]: weird.

**Review by nilsding — APPROVED** [review]

**Review by pixeldesu — APPROVED** [review]
Tested through the application, everything still seems to work fine! LGTM 👍

**Pull request merged**